components/admin/modules/SideBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import _ from 'lodash'
+import { Module } from '../../../graphql/index'
+import styles from '../../../scss/sideBar.module.scss'
+import { XIcon, ChevronRightIcon } from '@primer/octicons-react'
+
+type SideBarProps = {
+  modules: Module[] | []
+  selectedModule: number
+}
+export const SideBar: React.FC<SideBarProps> = ({
+  modules,
+  selectedModule
+}) => {
+  const moduleListData = [_.cloneDeep(modules) || []]
+  const lessonList = moduleListData[0].map((obj: any, arrIndex: number) => (
+    <>
+      <a
+        key={'module' + obj.id}
+        data-testid="module-title"
+        role="tab"
+        className={
+          `${styles.sidebar_tabsNav}` +
+          (arrIndex === selectedModule
+            ? ` ${styles.sidebar_tabsNav__active}`
+            : '')
+        }
+        data-toggle="pill"
+        style={{ wordBreak: 'break-word', cursor: 'pointer' }}
+      >
+        <div className="innerText">{obj.name}</div>
```

Comment:
This should be a `span`, `hX` element or `Text` from our [components](https://github.com/garageScript/c0d3-app/blob/2d5961899002c4bba480e5c04971ee8e4b205554/components/theme/Text.tsx). 

OR

Wrap the `{obj.name}` with one of the above elements. E.g., `<div ...> <Text>{obj.name}</Text> </div>`